UI/js/signup.js

```diff
@@ -1,7 +1,6 @@
 const form = document.getElementById("signUp");
 const statusDiv = document.getElementById("status");
 
-
 const toJSONString = formhtml => {
   const obj = {};
```

Comment:
If this function and the above `toJsonString` function are doing the same thing it is better to create a helper class, add the function there then call it from that file for all classes that will be using it. So your code can be [DRY](http://wiki.c2.com/?DontRepeatYourself)